GPipe-GLFW/GPipe-GLFW.cabal

```diff
@@ -49,4 +49,4 @@ source-repository this
   type:     git
   location: https://github.com/plredmond/GPipe-GLFW.git
   subdir:   GPipe-GLFW
-  tag:      v1.4.1.4
+  tag:      v1.5.0.0
```

Comment:
You're adding a new field to the settings. I don't think that needs a minor version bump? Maybe a patch bump? Not sure.

Reply:
That's really up to to you. I've left it as it, but feel free to change it as you see fit.